### PR TITLE
Make connection specifically about GATT and define it better.

### PR DESCRIPTION
**Pull request by jyasskin**

### CODE REVIEW

**Comment by scheib** [comment]
(next on review list)

File: rationale.md [hunk 1]
```diff
@@ -78,3 +78,42 @@ and the [Android API](https://developer.android.com/reference/android/bluetooth/
 doesn't document which it prefers when both are available.
 Neither gives enough control to
 implement a web standard that gives the web page the ability to choose.
+
+## Why do we have just one connection instead of one per call to connect()?
+
+If each call to `BluetoothDevice.connect()` returned a separate `BluetoothGATTRemoteServer` instance,
```
Comment (scheib) [inline rationale.md]: .connect() -> .connectGATT().

**Comment by scheib** [comment]
LGTM

File: rationale.md (hunk 1 above)
Comment (jyasskin) [inline rationale.md]: Whoops, thanks!

**Pull request merged**